src/error.js

```diff
@@ -226,14 +226,13 @@ export function installErrorReporting(win) {
 
 /**
  * Signature designed, so it can work with window.onerror
- * @param {string|undefined} message
+ * @param {*|undefined} error
  * @param {string|undefined} filename
  * @param {string|undefined} line
  * @param {string|undefined} col
- * @param {*|undefined} error
  * @this {!Window|undefined}
  */
-function reportErrorToServer(message, filename, line, col, error) {
+function reportErrorToServer(error, filename, line, col) {
```

Comment:
@in installErrorReporting the function call has no parameter. All parameters are optional